test/watchCases/plugins/extract-text-plugin/0/index.js

```diff
@@ -0,0 +1,6 @@
+import css from "./style.css"
+
+it("should ignore change to file and directory", function() {
+	// nothing to assert
+	// this test would hang if stats.hash didn't update upon css change
```

Comment:
That's a bit too unsecure for me. We might change how watch cases are executed in future and this may make this test useless.

Let's wait until #3737 is merged so we could access the stats here and check the hash.